ui/app/package.json

```diff
@@ -6,6 +6,7 @@
     "@azure/msal-browser": "^2.24.0",
     "@azure/msal-react": "^1.4.0",
     "@fluentui/react": "^8.68.2",
+    "@fluentui/react-northstar": "^0.64.0",
```

Comment:
do we need this extra import? what is it used for? If it's for something not in `fluentui/react v8` then `v9` has been released, can we just bump the version and avoid the northstar import?